llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -7581,6 +7581,12 @@ Value *SPIRVToLLVM::transGLSLExtInst(SPIRVExtInst *extInst, BasicBlock *bb) {
     // Split input into fractional and whole number parts.
     Value *wholeNum = getBuilder()->CreateUnaryIntrinsic(Intrinsic::trunc, args[0]);
     Value *fract = getBuilder()->CreateFSub(args[0], wholeNum);
+    // Vectors are represented as arrays in memory, so we need to cast the array to a vector before storing.
```

Comment:
May be better to slightly modify the comment, "cast the pointer of array to pointer of vector" to match the code? And the issue may also applies to frexp(). But that is a separate change.

Reply:
The issue could happen for frexp. I will find a CTS assembly case to verify. I think it needs the same fix.